ui/public/locales/en.json

```
@@ -2880,6 +2904,8 @@
 "message.copy.iso.confirm": "Please confirm that you wish to copy your ISO to",
 "message.copy.template": "Copy template <b id=\"copy_template_name_text\">XXX</b> from zone <b id=\"copy_template_source_zone_text\"></b> to",
 "message.copy.template.confirm": "Are you sure you want to copy template?",
+"message.cpu.usage.info.multi.cpu": "The percentage of CPU usage can go over 100% when a VM has more than 1 vCPU. This happens because the usage of all the vCPUs of the VM are accumulated.",
```

@joseflauzino, actually it depends on how the hypervisor account the stats. As far as I know, the way VMware accounts it, the CPU will never pass 100%. I think a more accurately message would be:

```suggestion
"message.cpu.usage.info.multi.cpu": "The CPU usage percentage can exceed 100% if the VM has more than 1 vCPU or when CPU Cap is not enabled. This behavior happens according to how the hypervisor being used accounts the stats.",
```